src/containers/stage.jsx

```diff
@@ -95,6 +95,10 @@ class Stage extends React.Component {
         }
         this.updateRect();
         this.renderer.resize(this.rect.width, this.rect.height);
+        if (this.props.isFullScreen !== prevProps.isFullScreen) {
```

Comment:
Would this be better as `this.props.stageSize !== nextProps.stageSize`?

There's another bug where:
1. You switch to small stage mode
2. You switch from the editor to the project page
3. The stage appears low-resolution